makefile.shade

```diff
@@ -463,4 +464,17 @@ functions
                         .Replace("\n", "|n")
                         .Replace("]", "|]");
         }
+
+        string GetBuildBranch() 
```

Comment:
At the top of the file:

`default BUILD_BRANCH = '${Environment.GetEnvironmentVariable("BUILD_BRANCH") ?? "dev"}``